docs/source/projects.rst

```diff
@@ -348,7 +400,7 @@ Environment
     By default, MLflow Projects are run in the environment specified by the project directory
     or the ``MLproject`` file (see :ref:`Specifying Project Environments <project-environments>`).
     You can ignore a project's specified environment and run the project in the current
-    system environment by supplying the ``--no-conda`` flag.
+    system environment by supplying the ``--env-manager=local`` flag.
```

Comment:
Do you want to add a note or a warning here about ignoring version specifications can result in unstable or unpredictable prediction or inference results? Choosing to manually maintain the prediction environment and keeping it synchronized with the training environment is a bad idea without some form of environment automation and external handling of the CD process...

Reply:
Sounds good! I'll add it.